src/ActivityRepository.js

```diff
@@ -1,12 +1,13 @@
-class ActivityRepository {
-  constructor(id, activityData) {
+class ActivityRepository extends Stats {
+  constructor(id, data) {
+    super(id, data);
     this.id = id;
-    this.activityData = activityData;
-    this.user = this.getUserLogs(id);
+    this.data = activityData;
+    this.user = this.getUserLogs();
   }
 
-  getUserLogs(id) {
-    return this.activityData.filter(user => user.userID === id);
+  getUserLogs() {
```

Comment:
Can we remove these functions from the child classes since they can be accessed through the parent class?

Reply:
Make a comment here that includes what we talked about over Slack --

Let's keep this function in there for now, since we still have to cross reference the data files that we have locally.  Once we're fetching, we'll remove these and look into the fetched data that we have and pull from that to populate the DOM.